controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -120,6 +127,8 @@ def run_match(supervisor: Supervisor) -> None:
 def main() -> None:
     quit_if_development_mode()
 
+    sr_controller.tee_streams(REPO_ROOT / 'recordings' / log_filename())
```

Comment:
Is 'recordings' the best directory for this?

Reply:
Probably not. I'm open to ideas for better places to put it. I was mostly aiming for somewhere which was already somewhere that you might look when running competition mode but wasn't going to get in the way.